pg_search/src/api/search.rs

```diff
@@ -187,16 +212,23 @@ unsafe fn snippet(
     TableIterator::new(top_docs)
 }
 
-pub fn drop_bm25_internal(index_oid: pg_sys::Oid) {
-    // We need to receive the index_name as an argument here, because PGRX has
-    // some limitations around passing OID / u32 as a pg_extern parameter:
-    // https://github.com/pgcentralfoundation/pgrx/issues/1536
-
+pub fn drop_bm25_internal(database_oid: u32, index_oid: u32) {
     let writer_client = WriterGlobal::client();
+    let relfile_paths = WriterDirectory::relfile_paths(database_oid, index_oid)
+        .expect("could not look up pg_search relfilenode directory");
```

Comment:
The message style guide for `Result::expect()` suggests the message explain why the result should be `Result::Ok()`.  So I'd rewrite this message as something like:

> the relfilenode directory should exist

(or whatever that function does -- I haven't gotten there yet)